src/configurations/destinations/amazon_audience/ui-config.json

```diff
@@ -0,0 +1,300 @@
+                "note": "Update your connection settings here",
+                "icon": "settings",
+                "fields": [
+                  {
+                    "configKey": "advertiserId",
+                    "type": "nestedSelect",
+                    "label": "Select your Account",
+                    "apiName": "getAmazonAdvertiser",
+                    "placeholder": "Select Advertiser Id details",
+                    "preRequisites": {
+                      "fields": [
+                        {
+                          "configKey": "rudderAccountId",
+                          "exists": true
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "type": "singleSelect",
+                    "label": "Authorization servers",
+                    "note": " Support for more servers will be added soon",
+                    "configKey": "authServer",
+                    "options": [
+                      {
+                        "name": "North America",
+                        "value": "North America"
+                      }
+                    ],
+                    "default": "North America"
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "groups": [
+              {
+                "title": "Connection mode",
+                "note": [
+                  "Update how you want to route events from your source to destination. ",
+                  {
+                    "text": "Get help deciding",
+                    "link": "https://www.rudderstack.com/docs/destinations/rudderstack-connection-modes/"
+                  }
+                ],
+                "icon": "sliders",
+                "fields": []
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "title": "Configuration settings",
+        "note": "Manage the settings for your destination",
+        "sections": [
+          {
+            "groups": [
+              {
+                "title": "Optional configuration settings",
+                "note": "Configure advanced destination-specific settings here",
+                "icon": "settings",
+                "fields": [
+                  {
+                    "type": "checkbox",
+                    "label": "Enable user data hashing",
+                    "note": [
+                      "Enabling this will hash all the user data before sending it to Amazon Audience",
+                      "Do not enable it if you already pass hash data to rudderstack"
+                    ],
+                    "configKey": "enableHash",
+                    "default": false
+                  }
+                ]
+              },
+              {
+                "title": "Optional Audience Creation Configuration settings",
+                "icon": "settings",
+                "fields": [
+                  {
```

Comment:
why audienceId input is in destConfig ? shouldn't it be in VDM screen ?

Reply:
I see this is different field, externalAudienceId - how is it different from audienceId ?

Reply:
seems like  "externalAudienceId", "ttl", "dataSourceCountry", "enableHash" need to be part of audience creation screen ideally